packages/scoped-custom-element-registry/test/Element.test.html.js

```diff
@@ -55,4 +78,30 @@ describe('Element', () => {
       });
     });
   });
+
+  describe('insertAdjacentHTML', () => {
+    it('should upgrade an element defined in the custom registry', () => {
+      const {tagName, CustomElementClass} = getTestElement();
+      const registry = new CustomElementRegistry();
+      const shadowRoot = getShadowRoot(registry);
```

Comment:
OT: Do `getShadowRoot()` and `getHTML()` coordinate via shared global state? Makes for non-obvious flow here...

Reply:
No, you pass the root you want to innerHTML into into `getHTML`, and it just returns out the first element.